Makefile

```diff
@@ -52,14 +57,36 @@ deps.linux.amd64: duckdb
 .PHONY: deps.linux.arm64
 deps.linux.arm64: duckdb
 	if [ "$(shell uname -s | tr '[:upper:]' '[:lower:]')" != "linux" ]; then echo "Error: must run build on linux"; false; fi
+	mkdir -p deps/linux_arm64
 
 	cd duckdb && \
 	CC="aarch64-linux-gnu-gcc" CXX="aarch64-linux-gnu-g++" CFLAGS="-O3" CXXFLAGS="-O3" ${DUCKDB_COMMON_BUILD_FLAGS} make bundle-library -j 2
 	cp duckdb/build/release/libduckdb_bundle.a deps/linux_arm64/libduckdb.a
 
+.PHONY: deps.freebsd.amd64
+deps.windows.amd64: duckdb
+	if [ "$(shell uname -s | tr '[:upper:]' '[:lower:]')" != "mingw64_nt-10.0-20348" ]; then echo "Error: must run build on windows"; false; fi
+	mkdir -p deps/windows_amd64
+	
+	# this is just code copied from duckdb and fixed for windows. Would like to not change this, and use `make bundle-library` once its fixed.
+	cd duckdb && \
+	${DUCKDB_COMMON_BUILD_FLAGS} GENERATOR="-G \"MinGW Makefiles\"" BUILD_JSON=1 gmake release -j 2
```

Comment:
I am undecided about compiling `JSON` into the build, as it diverges from the other builds... cc @marcboeker

Reply:
The problem is that (At the time) there were no json extension builds for this bulid. (I even think there were no extensions available at all). So compiling it with duckdb was the only way to make json available to the users, and I found json to be essential enough to include it. Maybe we should try again when 1.1 has released, hopefully also including the header changes?

Reply:
Yes, that's a good call - we can pick this up after `1.1.` and discuss the extensions.